src/dune_action_plugin/glob.ml

```diff
@@ -0,0 +1,19 @@
+open Stdune
+
+type t =
+  { re : Dune_re.re
+  ; repr : string
+  }
```

Comment:
Why does the action needs its own module for this? Can't we just move this to a `dune_glob` library (that would also include the lexer)?

Reply:
I think yes, it should move to a dune_glob library. We talked about that with Jakub, but never got around to making the change.

Reply:
I pushed the commit cfe1d89 that creates a dune-glob package.